chapters/ch10.md

```diff
@@ -1,23 +1,25 @@
 # Having a dev set and metric speeds up iterations
 ->
-
+# Có một tập phát triển và một phép đo sẽ thúc đẩy nhanh quá trình làm việc.
```

Comment:
metric nên dịch là độ đo. 'thúc đẩy vòng lặp thử nghiệm mô hình'

Reply:
Em cũng thấy độ đo hay hơn nhưng theo glossary phải dịch là phép đo anh ạ :((

Reply:
how about 
"(Xây dựng/Lựa chọn) Có tập phát triển và phép đo giúp thúc đẩy tiến trình"

Reply:
Done anh @ngcthuong!

Reply:
phép đo thì được thực hiện đi thực hiện lại. còn độ đo thì chỉ có 1 độ đo cụ thể cho bài toán. trong câu này metric là số ít, em ko nghĩ là để phép đo ở đây thì hợp lý.

Reply:
Theo ý mình hiểu là 
"cần định nghĩa tập phát triển và "phép đo ---> trả về trị số" để đánh giá chất lượng ---> từ đó thúc đẩy nhanh cả quá trình phát triển thuật toán. 

Phép đo này sẽ thực hiện đi thực hiện lại như phần nội dung của bài khi nói về vòng lặp của tiến trình. 

Nên anh vẫn nghĩ để **phép đo** hợp lý hơn hơn là "độ đo".

Reply:
@xuantubka hiện có một yêu cầu là tất cả các từ được dịch phải theo `glossary.md`.  Hiện từ đầu đã được dịch là "phép đo" nên mình giữ nguyên một cách dịch cho nhất quán.

Mọi người vẫn có thể đề xuất cách dịch khác ở một Pull Request riêng. Nếu thấy cách dịch mới hợp lý thì sẽ tạo thêm một commit để thay đổi toàn bộ các chương đã hoàn thành có chứa từ đó.

Em có thể tạo một PR để mọi người bàn về việc này. Anh sẽ link vào bảng thuật ngữ để sau này nếu từ đó được chấp nhận thì tại sao chúng ta lại dịch như vậy.

Anh từng thấy "độ đo" được sử dụng trong các tài liệu toán, như nó được dịch từ "measure" trong tiếng Anh, như trong bài [wikipedia này](https://www.google.com/search?q=độ+đo&ie=utf-8&oe=utf-8&aq=t).